stacks/services.yml

```diff
@@ -18,6 +18,7 @@ services:
       - io.docksal.permanent=${SANDBOX_PERMANENT:-false}
     environment:
       - APACHE_DOCUMENTROOT=/var/www/${DOCROOT:-docroot}
+      - APACHE_FCGI_HOST_PORT=cli:9000
```

Comment:
Whats this?

Reply:
This tell apache where PHP-FPM is. `cli:9000` is set by default, but with the upcoming changes for php/cli images this will change. Exposing it here for better visibility.